Tests/APITesters/SwiftAPITester/SwiftAPITester/PurchasesAPI.swift

```diff
@@ -299,13 +299,16 @@ func checkNonAsyncMethods(_ purchases: Purchases) {
 private func checkConfigure() -> Purchases! {
     Purchases.configure(with: Configuration.Builder(withAPIKey: ""))
     Purchases.configure(with: Configuration.Builder(withAPIKey: "").build())
+    Purchases.configure(with: Configuration.Builder(withAPIKey: "")
+        .with(observerMode: true, storeKitVersion: .default)
+        .build())
     Purchases.configure(with: Configuration.Builder(withAPIKey: "")
         .with(showStoreMessagesAutomatically: false)
         .build())
 
     Purchases.configure(withAPIKey: "")
     Purchases.configure(withAPIKey: "", appUserID: nil)
-    Purchases.configure(withAPIKey: "", appUserID: nil, observerMode: true, storeKitVersion: .storeKit2)
+    Purchases.configure(withAPIKey: "", appUserID: nil, observerMode: true, storeKitVersion: .default)
```

Comment:
almost wonder whether having `.default` be an option for this specific API is dangerous and we should limit the options to 1 vs 2 with a different enum